### PR TITLE
Felinids don't like getting wet

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
Blame @uomo91 

## Why It's Good For The Game
More downsides for felinids.

## Changelog
:cl:
add: Felinids now have a negative moodlet instead of a positive one from showering.
add: towels can now be crafted from cloth stacks and used to dry yourself or others.
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
[the new felinid experience](https://www.youtube.com/watch?v=xQjg9_pZnSE)

**Comment by SkeletalElite** [comment]
Should apply when being sprayed with with a spray bottle backpack sprayer

**Comment by Tharcoonvagh** [comment]
Make it stack

File: code/__DEFINES/dcs/signals.dm
Comment (Qustinnus) [inline code/__DEFINES/dcs/signals.dm:250]: bitshift pls

File: code/datums/mood_events/needs_events.dm
Comment (Qustinnus) [inline code/datums/mood_events/needs_events.dm:96]: 😏

**Comment by spookydonut** [comment]
Added towels

**Comment by Dorsisdwarf** [comment]
did this get feature approval

**Comment by Shadowflame909** [comment]
This one is probably gonna get denied, like the **majority** of other things that tried to add features to felinids.

Current maintainer policy is that, any felinid change that has a tiny change of causing issues, will be ripped out and devoured by the void.

**Comment by wesoda25** [comment]
bruh so all I need is a fire extinguisher to ruin someones day? i like it

**Comment by Iatots** [comment]
Even with magic fire extinguishers that spray distilled water, would that not be unpleasant?

**Comment by itseasytosee** [comment]
_Orange is coming_

**Comment by spookydonut** [comment]
brah i'm a maintainer do you think i'd have made this pr if it wasn't ok?

**Comment by interestingusernam3** [comment]
Slightly unfortunate title

**Comment by ATH1909** [comment]
IMO, this should happen in, say, the code for reaction_mob() for the water reagent, not just in the code for showers, so that you can spritz a felinid with a spray bottle of water to reprimand them.

**Comment by Qustinnus** [comment]
showers dont mob react

**Comment by kevinz000** [comment]
:+1:

**Comment by Acer202** [comment]
Could you... please balance it with an upside at the very least? Maybe a quirk you could take to offset it? A constant, never-ending train of downsides wears one down. I like to keep my character clean and blood-free. Why should I be punished for that? Not all cats hate water, after all.

**Comment by Qustinnus** [comment]
humans dont have downsides so you are in luck

**Comment by Acer202** [comment]
> humans dont have downsides so you are in luck

I really wish we didn't have this culture of bullying, but I suppose there's not much I can do. I'm willing to bet the vast majority of support on this comes from people who don't play as a felinid. Please consider how this discourages keeping characters clean for purely stereotypical reasons. Cats that love water are fairly common. Make it toggleable in some form for RP reasons at least? My character suddenly not liking showers doesn't make sense.

https://www.youtube.com/watch?v=PAH2vvVHXFg



> IMO, this should happen in, say, the code for reaction_mob() for the water reagent, not just in the code for showers, so that you can spritz a felinid with a spray bottle of water to reprimand them.

Wouldn't that effect then happen by drinking a glass of water, too?

**Comment by itseasytosee** [comment]
I really wish felinids were not in the game at all because I don't think it fits the theme and tends to attract people who I often don't want to play with. But we can't all get our wishes huh.

**Comment by Acer202** [comment]
> I really wish felinids were not in the game at all because I don't think it fits the theme and tends to attract people who I often don't want to play with. But we can't all get our wishes huh.

Felinid-like races have been in Star Trek for longer than I, and likely you, have been alive. It's hard to argue they aren't thematically fitting in a game that "borrows" so much from space pop culture and has no solid lore commitments. This affects the people you _do_ like to play with who happen to play the race too, eh? We shouldn't be approving content out of spite for others. That's a sign of a toxic community.

**Comment by wesoda25** [comment]
lol

**Comment by Qustinnus** [comment]
did you just compare caitians to anime cat girls? truly a daring comparision

**Comment by tralezab** [comment]
hey guys if you like this pr you'd LOVE felinid genetic wolverine claws

**Comment by RaveRadbury** [comment]
If they're going to hate showers could they maybe get more of a boost from freshly laundered clothing?

**Comment by itseasytosee** [comment]
@Acer202 frankly I don't care about this content and I'm certainly not supporting it to spite anyone. The downside negligible considering how hygiene was tossed away a while ago and you can clean your clothing just about anywhere there's metal. Debates about this sort of thing are dumb considering how nothing is ever going to change.

**Comment by Acer202** [comment]
> @Acer202 frankly I don't care about this content and I'm certainly not supporting it to spite anyone. The downside negligible considering how hygiene was tossed away a while ago and you can clean your clothing just about anywhere there's metal. Debates about this sort of thing are dumb considering how nothing is ever going to change.

Why are you supporting it, if I may ask? As someone who plays as a felinid, especially as one who claims to like swimming and staying clean, (a bloody jumpsuit and a lack of care for bloody shoes is pretty absurd from an RP standpoint) this is a disheartening change forced upon me for purely stereotypical reasons. With this, I'll just continue to RP that my character likes showers while dealing with another unchallenged nerf. This change does essentially nothing for an outward observer. Laser pointer interaction is quirky and I see people have fun with that all the time, but this is purely an invisible debuff that very much feels out of spite based on the reaction I get here and the lack of conversation about it. It forces me to RP against gameplay mechanics added for little to no reason. Please be considerate with how this acts as an annoyance to players. If you want your felinid to hate water, just roleplay it out.

**Comment by Qustinnus** [comment]
just use laundry

**Comment by itseasytosee** [comment]
That's a copypasta and a half yikes.
I don't want to antagonize you any further as you are clearly quite passionate about this, but to answer your question I gave this or a thumbs up because I thought it was funny.

**Comment by Acer202** [comment]
> That's a copypasta and a half yikes.
> I don't want to antagonize you any further as you are clearly quite passionate about this, but to answer your question I gave this or a thumbs up because I thought it was funny.

Just please encourage the RP in other ways, then. Don't force nerfs on people because you think it's funny. This is definitely a feelsbad PR for me, and I'm sorry for going on like I have.

**Comment by itseasytosee** [comment]
If it makes you feel any better I changed my thumb up to a laugh <3

**Comment by triplezeta** [comment]
yall out here really arguing over a moodlet and cat ears smh my head

**Comment by Acer202** [comment]
> yall out here really arguing over a moodlet and cat ears smh my head

The problem is that it has actual gameplay consequences and forces an awkward situation when you RP that you do indeed like water. Being punished for wanting to keep a clean character is weird. Now I'm forced to strip down in dorms to avoid getting a debuff? Why? Please no.

**Comment by CRITAWAKETS** [comment]
just make hydrophobia a quirk, so that everyone can have it and felinids aren't forced to have it

problems solved

**Comment by CRITAWAKETS** [comment]
> yall out here really arguing over a moodlet and cat ears smh my head

gotta love the figurehead of the moth metagang complaining about felinids

**Comment by triplezeta** [comment]
> complaining

**Comment by zxaber** [comment]
Acer202, just make a towel and dry yourself afterwards. That removes the debuff.

**Comment by Acer202** [comment]
> Acer202, just make a towel and dry yourself afterwards. That removes the debuff.

But why even add this at all? All it does is make trying to stay clean detrimental and adds no enjoyment. It just seems in bad faith,

**Comment by wesoda25** [comment]
I mean personally I think the absurdity of someone who hates water but loves being clean is poetic, congrats you have a new gimmick.

**Comment by zxaber** [comment]
It adds some flavor to the felind race to help seperate them from being just humans with cat ears. Little differences like this are neat and should be encouraged for all races (like how moths have issues welding without double eye shielding).

**Comment by shellspeed1** [comment]
> 
> 
> It adds some flavor to the felind race to help seperate them from being just humans with cat ears. Little differences like this are neat and should be encouraged for all races (like how moths have issues welding without double eye shielding).

What about every other attempt to give felenids flavor, like the claw mutation?

**Comment by Iatots** [comment]
The janitor can now adopt the smelly cats.
What if they got a "brushy brushy" mood buff from being ran over by a janiborg/upgraded janicart?

**Comment by Acer202** [comment]
> I mean personally I think the absurdity of someone who hates water but loves being clean is poetic, congrats you have a new gimmick.

My gimmick is trying my best to be a good spessman, as in not being a jerk and helping people whenever I can, and I don't want a new one. Getting punished because I want to stay clean is silly.

**Comment by Qustinnus** [comment]
if you don't want any gimmicks besides being a good guy you can play human, the race without gimmicks.

**Comment by Acer202** [comment]
> if you don't want any gimmicks besides being a good guy you can play human, the race without gimmicks.

Pardon me, but your reasoning sucks, You're implying that a different race _has_ to act a different way, Give me something better.

**Comment by Shadowflame909** [comment]
I offer you the very overpowered roundstart cat ear item in the theatre vendor.

Allows you to become a catgirl, and get this, without any of the downsides!

The cost is it requires you to not wear any other hat.

**Comment by Acer202** [comment]
> I offer you the very overpowered roundstart cat ear item in the theatre vendor.
> 
> Allows you to become a catgirl, and get this, without any of the downsides!
> 
> The cost is it requires you to not wear any other hat.

The constant wagging is really annoying. I don't like it. I'd also it not get in the way of using hardsuits when I'm an engi fixing the station.

**Comment by Shadowflame909** [comment]
wear it while augmented like an epic gamer robo cat girl

**Comment by itseasytosee** [comment]
All of this shit is pointless they are just humans who have had car ears graphed to there head why is this even a round start race.

**Comment by Acer202** [comment]
> wear it while augmented like an epic gamer robo cat girl

I don't understand why you support the idea of using kitty ears that make someone look identical to a race that you'd rather not they be. Why not just let them have a choice? Why try to make things harder for them?

**Comment by Shadowflame909** [comment]
the wolverine catgirl mutation denied.

so now i agree with maintainers that cat-girls are just a tatoo from the tatoo system. We're still waiting for someone to add in the rest of the tatoos.

**Comment by Acer202** [comment]
> All of this shit is pointless they are just humans who have had car ears graphed to there head why is this even a round start race.

It's a roundstart choice because we held a huge community-wide vote on the issue, and a majority of people were in favor of it being this way, probably because they enjoyed it. Is it so bad to have something others enjoy? I don't know why this has to be the subject of endless grief and short ends of the stick. I just wanna play my gosh darn catgirl in the spessmans game. Leave em' be.

**Comment by Qustinnus** [comment]
Yes, different races should be different. That's the point. They should offer changes in gameplay. What's the point of having different races if they're all the same?

You just said you don't want any gimmicks, so why do you want cat ears? That is a gimmick. Should we remove those from felinids?

**Comment by ishitbyabullet** [comment]
**Races are supposed to have a downside and or upsides while humans remain neutral. Lizards have very minor downsides (temperature bullshit), abductors have a downside which serves as an upside (z-level wide or beyond communication at the cost of being mute), plasmamen have upsides (hardcore mode race while also being skeletons technically), and golems have numerous downsides.**

If you play as a catperson but get incredibly fucking annoyed at the thought of having a downside then you are literally just trying to be a snowflake. This shouldn't even be a problem for you if you like playing as a catperson because this is EASY TO IGNORE. This isn't even as hardcore as plasmamen so unless you chose the quirk that makes you never able to recover sanity, you should EASILY be able to counter this.

**Comment by Acer202** [comment]
>you should EASILY be able to counter this.

But why add it at all? That's still the main problem. It just detriments people over something really minor. Most people don't bother trying to stay non-bloody, and as the one person who does, getting a little slap on the wrist for it is annoying. I'll RP running away when someone spray bottles me, but being forced to have a negative mood for a moment or stripping down in dorms when trying to maintain a believable character just seems silly to me, to the degree of annoyance. What joy does this bring to an outside observer? Is someone going to come in and laugh at me as I shower?

>Should we remove those from felinids?

Ears and tail are already toggleable, so that's an option if you wish.

**Comment by ishitbyabullet** [comment]
> But why add it at all? That's still the main problem. It just detriments people over something really minor. 

Because you can't really add any significant downsides without oranges denying the PR, therefore only super minor shit is allowed, I guess.

I don't think catpeople can even get worse hearing damage from flashbangs tbh so they probably have no downsides at all currently.

**Comment by Qustinnus** [comment]
You're ignoring my point. Either way there's no reason not to add this. Felinids are already the strongest non human species and this isn't even a debuff. Oh no I need to use a towel us felinid players have to suffer every time :(

**Comment by Acer202** [comment]
> I don't think catpeople can even get worse hearing damage from flashbangs tbh so they probably have no downsides at all currently.

Ah, felinid ears actually take way longer to return to normal hearing after an explosion or flashbang. I'd wager 2-3 times longer, based on when people caught in the same blast try to talk to me after some time, but I still can't hear them.

**Comment by itseasytosee** [comment]
I bet you play wendy in don't starve

**Comment by Acer202** [comment]
> Either way there's no reason not to add this.
I guess my biggest reason at this point is that I'm asking nicely, as someone who doesn't want a mood debuff on my character who openly admits to liking swimming and staying clean, because it feels silly and drives me bonkers that I would have to RP against gameplay mechanics made based off of stereotype and that we get another debuff with no balance to support it. It's not even a quirk that's visible to others—it brings no joy, like lasers do. Rave brought up a question about making something positive to differentiate them and was completely ignored. Please, just let me have this? Just leave it alone?



> I bet you play wendy in don't starve

No way, Wilson for life.

**Comment by Qustinnus** [comment]
it brings me joy

**Comment by Acer202** [comment]
But at what cost? The detriment of others?

**Comment by Qustinnus** [comment]
it's all worth it for the greater good

**Comment by Acer202** [comment]
And what's the greater good?

**Comment by ATH1909** [comment]
> showers dont use reagents

I was originally gonna say that you can just make this happen with both showering AND being splashed with water, but then I had an idea.

We make this only happen if a felinid (who is not on fire) is splashed or sprayed with water, and don't give the moodlet to felinids who stand under a shower (since that's usually voluntary, and some cats will indeed take baths on their own if they're given a source of water and not FORCED into the water). In addition, we increase the magnitude of the debuff from being sprayed with water as a felinid, but we make it last for only 30 seconds (or even 10 seconds). The felinid mains get to keep their characters clean/"cute"/not blood-soaked/not stinky, and the people who want to bully/torture felinids can pick up any ol' extinguisher off of the wall and spray them with it for being a "bad cat", giving the sprayed felinid(s) a funny negative moodlet that WON'T give them crippling depression for the next FIVE MINUTES.

Does this sound good to everyone?

**Comment by Acer202** [comment]
This sounds much better, and incentives more interactions, even. I can support that. Just make sure it can't be powergamed for hypnoflash usage and stuff, and it has my support.

**Comment by ATH1909** [comment]
> This sounds much better, and incentives more interactions, even. I can support that. Just make sure it can't be powergamed for hypnoflash usage and stuff, and it has my support.

Hypnotic flashes check sanity (and a few status effects), not mood, so it should be fine.

Making the hypnotic flash depend on mood instead of sanity might be an idea for the future, though... I bet that people could get really creative with that.

**Comment by itseasytosee** [comment]
sounds fun for a physicist item maybe?

**Comment by wesoda25** [comment]
do you think sppoky donut is afraid to check the comments on this pr

**Comment by SkeletalElite** [comment]
> But at what cost? The detriment of others?

literally just dry yourself with a towel or use the washing machine instead.

**Comment by cynic716** [comment]
as a catgirl main I agree with this pr as it is funny as fuck, but only if being sprayed at all causes the moodlet and has an emote that triggers when it happens

**Comment by spookydonut** [comment]
Showers and hygiene bots both use the same code for 'washing' so its an easy way to add it.

The whole reagent spraying thing is a mess because there's no real concept of something being 'wet' except for freshly mopped floors.

There's negative firestacks but that is not really the concept of wet, that's just "won't immediately burst into flames"

And as was pointed out, just towel yourself off, takes a few seconds.

Watch out for hygiene bots though.

**Comment by Acer202** [comment]
> And as was pointed out, just towel yourself off, takes a few seconds.

The absolute biggest concern for me is how this ultimately affects gameplay. The concept that felinids are especially adverse to being sprayed down is pretty funny, but doing it by means of a shower adversity is questionable. How will others enjoy this in a way that makes it worth anything aside from being a minor annoyance to felinid players? Are people going to drag felinids off to the showers to give them a minor mood debuff? If not, this is basically an invisible change that adds nothing good. The idea of fire extinguishers having the effect while showers do not is far preferable, as people will be much more likely to have fun with it, felinid or not. And most importantly, it won't feel like a punishment for keeping your character clean. Consider how others can have fun with this.

**Comment by Qustinnus** [comment]
its okay you can use laundry

**Comment by spookydonut** [comment]
> 
> 
> > And as was pointed out, just towel yourself off, takes a few seconds.
> 
> The absolute biggest concern for me is how this ultimately affects gameplay. The concept that felinids are especially adverse to being sprayed down is pretty funny, but doing it by means of a shower adversity is questionable. How will others enjoy this in a way that makes it worth anything aside from being a minor annoyance to felinid players? Are people going to drag felinids off to the showers to give them a minor mood debuff? If not, this is basically an invisible change that adds nothing good. The idea of fire extinguishers having the effect while showers do not is far preferable, as people will be much more likely to have fun with it, felinid or not. And most importantly, it won't feel like a punishment for keeping your character clean. Consider how others can have fun with this.

I think it should take more effort to soft grief felinids, eg, hygiene bot + splashing them with blood

**Comment by Acer202** [comment]
As is, this just makes it so I have to play myself to stay clean quickly. Do try to find a way to make the experience more than taking a shower with extra steps, eh? I can really get behind something others can enjoy like the laser pointer thing. Right now this idea doesn't make itself apparent to outward observers and just makes me take showers with extra steps, to nobody's amusement in practice. Make it fun for everyone!

**Comment by spookydonut** [comment]
> 
> 
> As is, this just makes it so I have to play myself to stay clean quickly. Do try to find a way to make the experience more than taking a shower with extra steps, eh? I can really get behind something others can enjoy like the laser pointer thing. Right now this idea doesn't make itself apparent to outward observers and just makes me take showers with extra steps, to nobody's amusement in practice. Make it fun for everyone!

You can towel off others, maybe someone else toweling you off can give you a positive moodlet

**Comment by Acer202** [comment]
Okay woah woah, I think I'd feel genuinely uncomfortable asking random people to "please dry me off with this towel." Perhaps not that. Maybe something like splashing a beaker of water on a felinid causes them to move a tile forward. This whole PR has felt like a bit of grief to me, honestly. It's why it bothers me so much. If it can be made into something that would see more interaction from others and not just punish voluntary showering, I'd be pleased, and very grateful.

**Comment by FancyCyclops** [comment]
You are nothing but a joke to them.

Your protests do nothing to halt the progress.

Quist will have his fetish content added.

**Comment by dinnerwoerror** [comment]
ok

i would like to compose my thoughts for a moment

but, to preface, speaking as an official Mentor of the Boomer Station 13 administration...

**Comment by itseasytosee** [comment]
80th comment

**Comment by anconfuzedrock** [comment]
can I just have the towels please

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
procs without docs

**Comment by ATH1909** [comment]
> The whole reagent spraying thing is a mess because there's no real concept of something being 'wet' except for freshly mopped floors.

Just add it to the reaction_mob() of water (touch and vapor delivery methods only, and only if they're not on fire). Bam, done.

> And as was pointed out, just towel yourself off, takes a few seconds.

It's still something you have to both know about and choose to actively do, though. Making the debuff last a much shorter period of time means that a felinid player who doesn't know that felinids, unlike every other race in the game, need to use a towel on themselves after showering (even after they've dried off, as represented by their fire stacks returning to 0) in order to not get incredibly depressed(,) isn't gonna get screwed over by a mechanic that probably only are going to interact with.

File: code/game/objects/items/towel.dm
Comment (Acer202) [reply code/game/objects/items/towel.dm:6]: If they will be added to the map, please put some in engi near showers so I don't get a debuff for not being a rad contamination hazard should the need arise!

**Comment by AdamElTablawy** [comment]
not sure if this pr is hilarious because of the salty cat or disturbing because qustinn is trying to brrrapcode again and force people to be stinky

**Comment by stylemistake** [comment]
Implement this moodlet for sprayed water. Limiting this to just showers is kinda pointless and misses out on a lot of interactions! ATH pointled out that sprayed water should add shorter, but stronger moodlets, and I agree.

**Comment by ThePainkiller** [comment]
Felinids should also be able to eat dead mice without being grossed out

File: code/game/objects/items/towel.dm
Comment (ATH1909) [inline code/game/objects/items/towel.dm:27]: this doesn't even reset your negative firestacks to 0

File: code/game/objects/items/towel.dm
Comment (ATH1909) [inline code/game/objects/items/towel.dm:21]: owo

**Comment by FancyCyclops** [comment]
@AdamElTablawy You should have been angry from the start.

This is fetish code under the mask of a felinid grief PR. Quist's kinks are slipping again.

The funny orange arrows flow once more.

File: code/game/objects/items/towel.dm
Comment (spookydonut) [reply code/game/objects/items/towel.dm:6]: imagine reading the pr 
https://github.com/tgstation/tgstation/pull/50635/files#diff-4ec5a6d8b25e1125fc4adec16395c0aaR297

**Comment by spookydonut** [comment]
This should address the issues that were raised.

Showers trigger water touch reaction so its now just a generic moodlet for water mob reaction

**Comment by Acer202** [comment]
> Showers trigger water touch reaction so its now just a generic moodlet for water mob reaction

Can we please not do the shower thing? That's my biggest issue with this as a regular engi player. I'll have to either store or make towels for engi because I don't want the debuff. Fire extinguishers are a fun idea, showers are a bother.

**Comment by HomocidlePancakes** [comment]
> Can we please not do the shower thing? That's my biggest issue with this as a regular engi player. I'll have to either store or make towels for engi because I don't want the debuff. Fire extinguishers are a fun idea, showers are a bother.

Or you can just eat a candy bar and feel happy again.

**Comment by spookydonut** [comment]
> 
> 
> > Showers trigger water touch reaction so its now just a generic moodlet for water mob reaction
> 
> Can we please not do the shower thing? That's my biggest issue with this as a regular engi player. I'll have to either store or make towels for engi because I don't want the debuff. Fire extinguishers are a fun idea, showers are a bother.

It already happens I'm not changing how showers work.

**Comment by spookydonut** [comment]
It's literally 30 seconds lol

**Comment by Acer202** [comment]
> It's literally 30 seconds lol

Aaaa not if I have to sit under it for 10 minutes because someone made a funny fusion can and dragged it past me!

**Comment by spookydonut** [comment]
> 
> 
> > It's literally 30 seconds lol
> 
> Aaaa not if I have to sit under it for 10 minutes because someone made a funny fusion can and dragged it past me!

I think you have your priorities mixed up

**Comment by HomocidlePancakes** [comment]
> Aaaa not if I have to sit under it for 10 minutes because someone made a funny fusion can and dragged it past me!

Rule whatever: Loosing is a part of the game.

You can also just pick a human or a xeno if showering in particular bother you that much.

**Comment by Acer202** [comment]
> I think you have your priorities mixed up

This whole thing just wears me down because I feel like I'm being punished for trying to not be a blood-drenched, contaminated shitter and it makes me sad. I dunno. I'm feeling defeated. Just pass this for the funnies, whatever.

**Comment by KetasDotepasto** [comment]
The reasons cats hate getting wet is because it stops their fur from keeping them warm. Cats living it hot areas on the other hand tend to like water. Since felinids don't have fur, this PR really doesn't make much sense.

But if we want to add flavor to felinids, why not give them a "groom self" verb, which gives them a similar buff as showering? Or how about social grooming? Felinids can groom themselves and others. Other felinids get a positive moodlet from being groomed, members of any other race get a negative moodlet ("Ehh, I just got licked by a felinid").

Then, a hairball mechanic could be added. After grooming something, felinids get a "potential hairball" status. The odds of a hairball actually forming would be determined by the lowest ammount of health during the duration of the "potential hairball" moodlet. So, if they remain perfectly healthy, they don't get a hairball. If they enter crit, a hairball always forms.

Once they have a hairball, they get a continually worsening moodlet. In order to get rid of the hairball, they have to cough. Maybe 10% chance of removing the hairball every time they cough. Once they cough up the hairball, they lose the "hairball" moodlet and get a "recently vomited hairball" moodlet, which expires after a while.

This would provide felinids with flavor, while also providing both an upside and a downside. The upside being good moodlet whenever they want it, the downside being the risk of hairballs.

**Comment by HomocidlePancakes** [comment]
>Post above

I summon the **PROHIBITION HAMMER**.

**Comment by KetasDotepasto** [comment]
What's wrong with felinids grooming themselves like a cat would?

**Comment by AtlasRules** [comment]
@KetasDotepasto no, just

n    o

**Comment by Tharcoonvagh** [comment]
lol

**Comment by Qustinnus** [comment]
@FancyCyclops not even my pr

File: code/__DEFINES/dcs/signals.dm
Comment (ExcessiveUseOfCobblestone) [inline code/__DEFINES/dcs/signals.dm:251]:
```suggestion
#define SPECIES_HATES_WATER 1									///flag to prevent nice shower moodlet
```

Comment (ExcessiveUseOfCobblestone) [reply code/__DEFINES/dcs/signals.dm:251]: cursed github

Comment (spookydonut) [reply code/__DEFINES/dcs/signals.dm:251]: its indented like all the other flag returns

**Comment by optimumtact** [comment]
This has remained open and attracted far too many comments.

Closing, and this stuff is getting on my nerves

**Comment by FancyCyclops** [comment]
So this is the only time you're not okay with anger?

**Comment by wesoda25** [comment]
Idk oranges some of the comments are pretty funny

**Pull request closed without merging**
